delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/utils/ClasspathResourceUtils.scala

```diff
@@ -21,10 +21,10 @@ trait ClasspathResourceUtils {
     * @return the content of the referenced resource as an [[InputStream]] or a [[ClasspathResourceError]] when the
     *         resource is not found
     */
-  def ioStreamOf(resourcePath: String): IO[ClasspathResourceError, InputStream] =
+  def ioStreamOf(resourcePath: String, classLoader: ClassLoader): IO[ClasspathResourceError, InputStream] =
```

Comment:
the classloader could be an implicit no ? As we should only use one in a given context